meter/tapo/connection.go

```diff
@@ -29,21 +14,13 @@ import (
 // https://k4czp3r.xyz/reverse-engineering/tp-link/tapo/2020/10/15/reverse-engineering-tp-link-tapo.html
 // https://github.com/fishbigger/TapoP100
 // https://github.com/artemvang/p100-go
-
-const Timeout = time.Second * 15
+// KLAP protocol reference implementation
+// https://github.com/insomniacslk/tapo
```

Comment:
Direkt nutzen kannst Du das nicht? Dann ließe sich die Wartung dafür auslagern ;)

Reply:
Hatte ich überlegt, aber dachte mir dann, dass wir nicht die volle Funktionalität brauchen. Hab die Struktur auch noch etwas umgruppiert... An dem Protokoll selbst wird sich denke ich, nicht so viel ändern.

Reply:
VS Code hat mir bei einem ersten Test auch immer das Projekt Repo automatisch aus dem Source Code gelöscht ...
```go
import (
  "github.com/insomniacslk/tapo"
)
```
Muss ich die in der go.mod evtl. manuell eintragen, falls ich das nutzen wollte?

Reply:
I can add a license to the project (BSD), so you can use it freely in case you change your mind in the future

Reply:
Was sind deine/unsere Anforderungen an eine externes Repo, um es in evcc einzubinden?

Reply:
@insomniacslk wir nutzen aktuell keine BSD Software sonder ausschließlich MIT. Mir ist nicht klar, wie wir die License Notice bei unseren Images einbinden sollten. Nutzung würde ich daher nach erster Einschätzung ausschließen.

Reply:
@andig I am not a lawyer, but it should be as simple as including a PDF mentioning that you / this group is not the sole owner of the copyright, and reference to this github repo with the list of external copyrights. However I can change it to MIT if it helps (the two licenses are compatible and their clauses are nearly the same, so I have no problems with changing it)

Reply:
I would be super happy with MIT. It fits the evcc license and has less restrictions.

Reply:
> Muss ich den Link in der go.mod evtl. manuell eintragen, falls ich das nutzen wollte?

Entweder das, oder

    go mod tidy

Reply:
> I would be super happy with MIT. It fits the evcc license and has less restrictions.

Done at https://github.com/insomniacslk/tapo/commit/52dbe7a83ca7dd0e680a14a3a74870fd5a45cbf9